### ISSUE TITLE
Staging Review finding: Missing focus indicators

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#113289]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IIR Team 
**Product:** Veteran Readiness and Employment (VR&E) aka Chapter 31 (28-1900)
**Feature:** Form Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

On each page of the form flow, focus is manually set (which is okay here), but the focus indicator is not visible.

A visible focus indicator lets people know where they are on the page, especially when using a keyboard. Without it, keyboard users can get lost and may not be able to use the site easily. Focus indicators help users see what is selected or active.

<details><summary>Link, screenshot or steps to recreate</summary>

Each step of the form flow.

<img width="1070" alt="Image" src="https://github.com/user-attachments/assets/38aa5c52-b44e-457f-bc50-a78b78175664" />

</details>

## Recommended action

Ensure the standard VA.gov focus indicator shows on the element that focus is being set to.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.4.7 Focus Visible - Level A](https://www.w3.org/TR/WCAG22/#focus-visible)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by kingbmike** [comment]
@rsmithadhoc  We have addressed this issue with [THIS ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1360/views/3?pane=issue&itemId=118013832&issue=department-of-veterans-affairs%7Cva-iir%7C1819), which in the comments you'll find that the Forms Team has [THIS ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4508) to complete the work for this from a more wholistic standpoint.

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) We have addressed this issue with [THIS ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1360/views/3?pane=issue&itemId=118013832&issue=department-of-veterans-affairs%7Cva-iir%7C1819), which in the comments you'll find that the Forms Team has [THIS ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4508) to complete the work for this from a more wholistic standpoint.

Thanks @kingbmike, good to close!

**Comment by kingbmike** [comment]
Thank you @rsmithadhoc
